File: MongotoPandas.py
import pymongo
import pandas as pd
from pymongo import MongoClient

# Connect to MongoDB
client = MongoClient("mongodb://localhost:27017/")
db = client["CALGARY"]  # Replace with your actual database name

# Query the collection and load data into DataFrame
collection = db["Statistics"]  # Replace with your collection name
data = pd.DataFrame(list(collection.find()))

# Display the DataFrame
print(data.head())

File: MongotoPandas.js
const { MongoClient } = require('mongodb');

async function run() {
  const client = new MongoClient("mongodb://localhost:27017/");

  try {
    await client.connect();
    const db = client.db("CALGARY");  // Replace with your database name
    const collection = db.collection("Statistics");  // Replace with your collection name

    const data = await collection.find().toArray();
    console.log(data);
  } finally {
    await client.close();
  }
}

run().catch(console.dir);
